### PR TITLE
Add support to Python 3.10

### PR DESCRIPTION
**Pull request by gamorales**
## Description

The purpose of this PR is to upgrade python 3.6 to 3.10 in django-cronman library.

## Downtime Check

- [x] Downtime check pass

### Quality Check stamps

  I tested it and it works:

  - [x] Guillermo Alfonso Morales
  - [x] Jhonatan da Rosa
  - [x] Danial Malik
  - [x] José Maria Dominguez

### How to test this

- Steps to Reproduce
    
    1. docker-compose run --rm django-cronman pytest
    2. Create a cron task in .../uh-adm/cron/crontask/
    
- Expected Results (After change)

- Previous results (Before change)

### Before deployment

  -

### After Deployment

  -

Related tier3 PR https://github.com/unhaggle/tier3/pull/6415

### CODE REVIEW

File: cronman/models.py [hunk 1]
```diff
@@ -8,8 +8,7 @@
 from django.conf import settings
 from django.db import models
 from django.utils import timezone
-
-import six
+from django.utils import six
```
Comment (danialmalik) [inline cronman/models.py:11]: It doesn't do us any benefit if we remove `six` from one source and add it from another source.
 lets remove it altogether if possible?

File: cronman/version.py [hunk 2]
```diff
@@ -3,4 +3,4 @@
 
 from __future__ import unicode_literals
 
-__version__ = "2.1.0"
+__version__ = "3.0.0"
```
Comment (danialmalik) [inline cronman/version.py:6]: There is already a published version 3.0.0. I think it wasn't updated here. 
image


So let's use 4.0.0.

**Review by danialmalik — CHANGES_REQUESTED** [review]
- Please add a link to the related tier3 PR where it can be tested.

File: cronman/version.py (hunk 2 above)
Comment (gamorales) [reply cronman/version.py]: Updated to 4.0.0

File: cronman/models.py (hunk 1 above)
Comment (gamorales) [reply cronman/models.py]: Six library deleted.

**Review by danialmalik — COMMENTED** [review]
While we're at it, could we please remove other occurrences of `six` as well?

**Review by danialmalik — APPROVED** [review]

**Review by rcichy — APPROVED** [review]

File: cronman/version.py [hunk 3]
```diff
@@ -3,4 +3,4 @@
 
 from __future__ import unicode_literals
 
-__version__ = "2.1.0"
+__version__ = "4.0.0b38dev0"
```
Comment (mbarakaja) [inline cronman/version.py:6]: Why is there a jump to 4.x here? Does that mean that it isn't backward compatible?

**Review by mbarakaja — CHANGES_REQUESTED** [review]
What about Travis? The tests must run in Travis with Python version from 3.6 to 3.10.

File: cronman/version.py (hunk 3 above)
Comment (danialmalik) [reply cronman/version.py:6]: Ah yes. My bad, didn't realize that the purpose was to keep it backward compatible :facepalm: 

@gamorales could you change the version? There is already a version 3.0.0 (being used in t3). So can you update this version to `3.1.0`?

**Review by mbarakaja — APPROVED** [review]

**Comment by danialmalik** [comment]
@gamorales since the PR is approved and ready to be merged. Could you please change the version from the current "Dev" version to a live one" `3.1.0` ?

**Comment by gamorales** [comment]
> @gamorales since the PR is approved and ready to be merged. Could you please change the version from the current "Dev" version to a live one" `3.1.0` ?

Done, should I update already tier3 library too?

**Comment by danialmalik** [comment]
@unhaggle/smooth-operators since you are guardian of this package? could you please merge the PR and release a new version? (and may be review the PR as well?)

Thanks guys!!

**Review by jhonatandarosa — APPROVED** [review]

**Pull request merged**